stage1/init/kvm/network.go

```diff
@@ -30,16 +30,16 @@ import (
 
 // GetNetworkDescriptions explicitly convert slice of activeNets to slice of netDescribers
 // which is slice required by GetKVMNetArgs
-func GetNetworkDescriptions(n *networking.Networking) []netDescriber {
-	var nds []netDescriber
+func GetNetworkDescriptions(n *networking.Networking) []NetDescriber {
```

Comment:
@jjlakis In Go we generally try to avoid `Get*` methods, see https://golang.org/doc/effective_go.html#Getters. It was an intuitive review comment, as I am still not used to the fact, that in rkt we do have a plethora of those Getters.

Leave it as is, as we have a separate issue for that: https://github.com/coreos/rkt/issues/2450.